Sources/FluentPostgresDriver/PostgresRow+Database.swift

```diff
@@ -1,13 +1,26 @@
-extension PostgresRow: DatabaseRow {
-    public func contains(field: String) -> Bool {
-        return self.column(field) != nil
+import class Foundation.JSONDecoder
+
+extension PostgresRow {
+    internal func databaseRow(using decoder: PostgresDataDecoder) -> DatabaseRow {
```

Comment:
This should be `decoder:` to match but probably fine since it's internal anyway.